### DISCUSSION TITLE
Missing spanish translations

### DISCUSSION THREAD

**Discussion by mzenz** [earthians/marley#562] [Q&A] [answered]
The healthcare app has many missing translations to Spanish which we'd like to be able to fix and contribute back to the application. Can someone please point us to where translation strings are configured?  As far as we can tell, they don't seem to be part of the application's repository.

**Comment by Sajinsr** [comment, marked as answer]
Hi @mzenz , Currently we dont have translations in this repo, but we'll get this fixed in a day or two and update you soon. Thanks for your interest to contribute, we will post you.
